angular-client/src/pages/charging-page/components/fault-display/fault-display.component.css

```diff
@@ -4,5 +4,5 @@
   align-items: left;
   justify-content: left;
   padding-left: 10%;
-  overflow: hidden;
+  overflow-y: auto;
```

Comment:
what's this change for?

Reply:
scrolling will work, to see all 50 displayed

Reply:
sounds good